src/projetos/SugestaoPage.tsx

```
@@ -152,7 +178,7 @@ export function SugestaoPage() {
                             </BreadcrumbSeparator>
                             <BreadcrumbItem>
                                 <BreadcrumbPage>
-                                    <span>Cadastro de Projetos</span>
+                                    <span>Sujestão de Projetos</span>
```

ESLint

[error] Replace `····································` with `··················`

(prettier/prettier)